src/DimpleB/Assignment4program13.java

```diff
@@ -0,0 +1,45 @@
+/* WAP to print output as below
+ Please enter string:
+Input:this is techno credits
+Output:siht is onhcet credits
+ */
+package DimpleB;
+import java.util.Scanner;
+
+public class Assignment4program13 {
+	void reverseevenindexstring(String str)
+	{
```

Comment:
Hi @DimpleBAug19 
Logic is correct..
but try to break every operation. it will be easy to understand you when you referring your code or other person.